### PR TITLE
added --salte-rating-disabled-color for colors under disabled mode

### PR DESCRIPTION
**Pull request by christophe-g**
Simple addition to have a different color when the component is on `disabled` mode.

### CODE REVIEW

File: salte-rating.html [hunk 1]
```diff
@@ -32,6 +33,10 @@
         color: var(--salte-rating-limit-color, rgba(200, 200, 200, 0.9));
       }
 
+      :host[disabled] .foreground-rating {
```
Comment (cecilia-sanare) [inline salte-rating.html]: Can you change `:host[disabled]` to `:host([disabled])`?

I just noticed that this selector isn't correct for Polymer 2+.

**Review by cecilia-sanare — CHANGES_REQUESTED** [review]
**Couple of minor Changes**

* Change all `:host[disabled]` to `:host([disabled])`. This will fix some selectors for Polymer 2+.
* Can you also add a small demo for the disabled state with a custom color?

Otherwise it looks good to me!

File: salte-rating.html (hunk 1 above)
Comment (christophe-g) [reply salte-rating.html]: Done - was not aware of `host[attribute]` not valid for Polymer 2.0. Still stuck with 1.0

Thanks for your rapid answer!

Comment (cecilia-sanare) [reply salte-rating.html]: Looks like you missed one of the `:host[disabled]` selectors.

Comment (christophe-g) [reply salte-rating.html]: pfff... sorry !

**Review by cecilia-sanare — APPROVED** [review]

**Comment by cecilia-sanare** [comment]
Thanks for your contribution @christophe-g!

I just published your changes in [2.1.0](https://github.com/salte-io/salte-rating/releases/tag/2.1.0).

**Pull request merged**